samples/TypeScript/promisified_cp.cts

```diff
@@ -0,0 +1,12 @@
+const { exec } = require("child_process");
+
+exports.run  = function run(command): Promise<{ stdout: string; stderr: string }> {
```

Comment:
`.cts` files should still use `import`/`export` syntax—they just get compiled to `.cjs` instead of `.js`.

Reply:
Oh thanks, I learned about it after opening this PR and then I forgot to update the example. I'll do it later today.